workspaces/adventure-pack/goodies/typescript/Number.prototype.chr/index.test.ts

```diff
@@ -0,0 +1,54 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "./index";
+
+describe("Number.prototype.chr", () => {
+  it("convert ASCII codepoint to ASCII string", () => {
+    expect(Number(65).chr()).toBe("A");
+    expect(Number(10).chr()).toBe("\n");
+    expect(Number(61).chr()).toBe("=");
+    expect(Number(126).chr()).toBe("~");
+    expect(Number(32).chr()).toBe(" ");
+    expect(Number(48).chr()).toBe("0");
+  });
```

Comment:
We can use arrays to make it more concise 😄. This also could apply to the rest of the tests!

`expect[Number(65).chr(), Number(10).chr() ...]).toStrictEqual(["A", "\n" ...]);`

Reply:
I would push back on this -- taken to the extreme, all of the assertions in the file could be combined into a single one using an array of conditions, but what we gain in conciseness I think could also be a loss of clarity.

I think if we were to use `it.each` as suggested below, that could give us the best of both worlds.